packages/features/portfolio/src/hooks/use-vault-holdings-table.tsx

```diff
@@ -479,7 +481,7 @@ export const useVaultHoldingsTable = () => {
     }
   }, [expandedRows, setExpandedRows]);
 
-  if (totalVaultHoldings) {
+  if (totalVaultHoldings && !totalVaultHoldings.amountPaid.isZero()) {
```

Comment:
I don't think you can rely on this as a filter if the subgraph is down, you should actually add something that counts the total positions inside the holdings